tests/test_docs.py

```diff
@@ -166,6 +166,7 @@ def set_cwd():
 
 
 @pytest.mark.filterwarnings('ignore:(parse_obj_as|schema_json_of|schema_of) is deprecated.*:DeprecationWarning')
+@pytest.mark.filterwarnings('ignore:This functionality is experimental:UserWarning')
```

Comment:
It could be worth documenting this interaction with pytest & warnings (and runtime) if users are actually wanting to use it for anything more than scripts.

Reply:
I think this falls into the general "experimental" category. How about we make a `PydanticExperimental` warning and document that @sydney-runkle ?

Reply:
That sounds good to me! I can do that if you'd like!